lib/common/tests/agents/pcmk_get_ra_caps_test.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include <crm/common/agents.h>
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+static void
+ocf_standard(void **state) {
+    uint32_t expected = pcmk_ra_cap_provider | pcmk_ra_cap_params |
+                        pcmk_ra_cap_unique | pcmk_ra_cap_promotable;
+
+    assert_int_equal(pcmk_get_ra_caps("ocf"), expected);
+    assert_int_equal(pcmk_get_ra_caps("OCF"), expected);
```

Comment:
BTW This function should probably use case-sensitive comparisons, but I'd be concerned about backward compatibility